shell/process.go

```diff
@@ -44,6 +45,23 @@ func (p *Process) Run() error {
 		cmd.Dir = p.Command.Dir
 	}
 
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt,
+		syscall.SIGHUP,
+		syscall.SIGTERM,
+		syscall.SIGINT,
+		syscall.SIGQUIT)
+
+	go func() {
+		sig := <-signals
+		if cmd.Process != nil {
+			//fmt.Printf("Passing signal %v to sub-process %d\n", sig, cmd.Process.Pid)
+			cmd.Process.Signal(sig)
+		}
+	}()
+
+	defer signal.Stop(signals)
```

Comment:
I don't think signal stop will close the channel, and if the channel doesn't get closed then the goroutine will leak... but then I guess it goes (hehe) away when the bootstrap dies?